.github/workflows/ci.yml

```diff
@@ -26,6 +26,11 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
```

Comment:
why wasn't this necessary before?

Reply:
Before these changes, we did not have a Node version specified. This PR fixes the node version in the `package.json` and also adds a `.nvmrc` file.

The [ubuntu image](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md) that Github Actions uses already includes a Node version by default, so we were using that one, but right not it doesn't match with the one we set and that's why we need to install it manually

Reply:
ahhh nice